openprescribing/web/templates/web/base.html

```diff
@@ -0,0 +1,49 @@
+{% load static %}
```

Comment:
Very minor nit, but give that we may be stuck with this for a while I think it's worth raising: having `web/templates/web` feels annoyingly redundant to me. Could we add `openprescribing/web/templates` as an explicit path in TEMPLATE_DIRS to avoid this?

Reply:
I think that would be my preference. So `openprescribing/web/templates/web/base.html` moves to `openprescribing/web/templates/base.html` and then `settings.TEMPLATE_DIRS` has whatever it needs to make that work.